pkgs/applications/networking/calls/default.nix

```diff
@@ -62,17 +68,12 @@ stdenv.mkDerivation rec {
     xvfb_run
   ];
 
-  mesonFlags = [
-    # docs fail to build
-    # https://source.puri.sm/Librem5/calls/issues/99
-    "-Dgtk_doc=false"
```

Comment:
Actually, I left the `false` here even thought it is the default value so that we remember to enable it once the issue is fixed. Which seems to be the case now.

Would need to add `"devdoc"` to `outputs` attribute. And `gtk-doc` and `docbook-xsl-nons` and one of `docbook_xml_dtd_XY` (where `XY` is e.g. `43`, based on the docbook warnings in the build log) to `nativeBuildInputs`.

Reply:
I followed this and enabled the docs. I checked the warnings, it was 43. It also needed glib(gio) to build.